### ISSUE TITLE
[ML] Modify Classification to use Pipeline

### ISSUE THREAD

**Issue by w11wo** [bookbot-kids/speechline#44]
To better unify the API design, re-implement batched audio classification via HuggingFace `pipeline`. Noting equivalence in logits, preprocessing, and runtime speed.

- [x] Source Code
- [x] Test Suites
- [x] Formatting
